### ISSUE TITLE
[2.0.0-beta.2] Unfriendly error when user tries to create a project before nitro init

### ISSUE THREAD

**Issue by tomdavies** [craftcms/nitro#237]
### Description

If the user runs `nitro create project-name` before having run `nitro init`, the error message shown is somewhat unfriendly:

```
Error: there is no config file for the environment
```

Also, the check only happens after the user selects a PHP version.

Would be great to check for `~/.nitro/nitro.yaml` at the start of `nitro add` / `nitro create` and show a friendlier error, along the lines of:

```
Error: there is no config file for the environment
Did you forget to run `nitro init`
```

### Additional info

- Nitro version: 2.0.0-beta.2
- Docker version: 3.1.0
- MacOS Mojave 10.14.6
- Installed nitro via homebrew

**Comment by jasonmccallister** [comment]
@tomdavies just pushed a fix for this, it will prompt you to run `nitro init` and execute the command for you (similar to `nitro apply` prompts). Will be pushed out in the next beta. Thank you!

![CleanShot 2021-02-05 at 10 12 50@2x](https://user-images.githubusercontent.com/5354908/107051808-e20c1800-679a-11eb-9eda-35aa6aa075d9.png)

**Comment by jasonmccallister** [comment]
This was released with 2.0.0-beta.3. Thank you for the report!
